.github/workflows/spec.yml

```
@@ -54,10 +54,10 @@ jobs:
           buildevents step $TRACE_ID $STEP_ID $STEP_START 'Setup Environment'
           echo STEP_ID=Setup-Acceptance-Test-Matrix >> $GITHUB_ENV
           echo STEP_START=$(date +%s) >> $GITHUB_ENV
-#       - name: Run Static & Syntax Tests
-#         if: ${{ github.repository_owner == 'puppetlabs' }}
-#         run: |
-#           buildevents cmd $TRACE_ID $STEP_ID 'static_syntax_checks' -- bundle exec rake syntax lint metadata_lint check:symlinks check:git_ignore check:dot_underscore check:test_file rubocop
+      - name: Run Static & Syntax Tests
+        if: ${{ github.repository_owner == 'puppetlabs' }}
+        run: |
+          buildevents cmd $TRACE_ID $STEP_ID 'static_syntax_checks' -- bundle exec rake syntax lint metadata_lint check:symlinks check:git_ignore check:dot_underscore check:test_file rubocop
```

I don't really understand why this is so complex. It can be this easy:
```suggestion
          buildevents cmd $TRACE_ID $STEP_ID 'static_syntax_checks' -- bundle exec rake validate lint check rubocop
```

I also submitted a PR for this in https://github.com/puppetlabs/pdk-templates/pull/441 but it looks like this is lost somehow?